packages/traceability-schemas/schemas/CargoItem.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$id": "https://w3id.org/traceability/schemas/CargoItem.json",
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "$comment": "{\"term\": \"CargoItem\", \"@id\": \"https://service.unece.org/trade/uncefact/vocabulary/uncefact/#ConsignmentItem\"}",
+  "title": "Cargo Item",
+  "description": "Addresses the cargo items to be stuffed into a container for a shipment. A cargo item cannot be split across containers.",
+  "type": "object",
+  "properties": {
+    "@context": {
+      "type": "array"
+    },
+    "type": {
+      "oneOf": [
+        {
+          "type": "string"
+        },
+        {
+          "type": "array"
+        }
+      ]
+    },
+    "cargoLineItems": {
+      "$comment": "{\"term\": \"cargoLineItems\", \"@id\": \"https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.1#/components/schemas/cargoLineItem\"}",
```

Comment:
These id's still don't feel right to me, but I think we can refine incrementally.

Reply:
I completely agree, but have nothing better to point them at. My plan is that showing the intended usage, we can work with DCSA to improve this.